pygen/pygen_src/isa/riscv_compressed_instr.py

```diff
@@ -31,54 +31,54 @@ def __init__(self):
     @vsc.constraint
     def rvc_csr_c(self):
         # Registers specified by the three-bit rs1’, rs2’, and rd’
-        with vsc.implies(self.format.inside(vsc.rangelist(riscv_instr_format_t.CIW_FORMAT,
+        with vsc.if_then(self.format.inside(vsc.rangelist(riscv_instr_format_t.CIW_FORMAT,
```

Comment:
just wondering, was there some issue with using the `implies` constraints?

Reply:
Hi @udinator 
No, back then there was some issue with the if_then constructs. I have changed this Just to be consistent with the if and else statement.